src/routes/login/+page.svelte

```diff
@@ -24,6 +27,18 @@
       'You are not authorized to access the page that you attempted to view. Please contact a tool administrator to request access.';
   }
 
+  $: if (reason) {
+    if (reason.includes(EXPIRED_JWT)) {
+      error = 'Your session has expired.';
+      fullError = 'Your session has expired. Please log in again.';
+    } else {
+      error = decodeURIComponent(reason);
+      fullError = null;
+    }
+
+    removeQueryParam('reason');
```

Comment:
Getting a `ReferenceError: window is not defined` in `removeQueryParam` when going to the login page directly and pasting:

http://localhost:3000/login?reason=UhO

Edge case, but the SSR is tanking. Can we add a "browser" check to `removeQueryParam`?

Reply:
Ah got it, yep I noticed that other bug you filed that's causing SSR issues for likely the same reason https://github.com/NASA-AMMOS/aerie-ui/issues/718. I was already thinking we should just add the `browser` check inside the removeQueryParam method, then it should automatically fix that one too right?

Reply:
Ah yes! Thanks for posting that issue I totally for got about it. Yeah we can close two issues in one if you don't mind adding the check!

Reply:
🔥